src/Controller/PageAdminController.php

```diff
@@ -101,7 +101,11 @@ public function treeAction(Request $request = null)
         if (!method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
             $twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')->renderer->setTheme($formView, $theme);
         } else {
-            $twig->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')->setTheme($formView, $theme);
+            // BC for Symfony < 3.4 which deprecated this runtime in favor of FormRenderer
+            if (!method_exists('Symfony\Bridge\Twig\Command\DebugCommand', 'getLoaderPaths')) {
```

Comment:
Please use the `::class` notation as much as you can.

Reply:
It would be easier to just copy the method from AdminBundle